package.json

```diff
@@ -4,66 +4,66 @@
   "private": true,
   "dependencies": {
     "@date-io/date-fns": "^1.3.11",
-    "@date-io/moment": "^2.0.1",
-    "@material-ui/core": "^4.8.3",
-    "@material-ui/icons": "^4.5.1",
-    "@material-ui/lab": "^4.0.0-alpha.42",
+    "@date-io/moment": "^2.10.8",
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.57",
     "@material-ui/pickers": "^3.2.8",
-    "caniuse-lite": "^1.0.30001021",
-    "chart.js": "^2.9.3",
+    "caniuse-lite": "^1.0.30001192",
+    "chart.js": "^2.9.4",
     "classnames": ">=2.2.5",
-    "cropperjs": "^1.5.6",
-    "date-fns": "^2.6.0",
-    "downshift": "^4.0.6",
+    "cropperjs": "^1.5.11",
+    "date-fns": "^2.17.0",
+    "downshift": "^4.1.0",
     "history": "^4.10.1",
     "http-proxy-middleware": "^0.20.0",
-    "interweave": "^12.1.1",
-    "jwt-decode": ">=2.2.0",
-    "lodash": ">=4.17.11",
-    "luxon": "^1.21.3",
-    "moment": ">=2.24.0",
+    "interweave": "^12.7.0",
+    "jwt-decode": "^3.1.2",
```

Comment:
I just want to point out the change in major version here and want to make sure there were no breaking changes...

Reply:
The only breaking change from v2 -> v3 for jwt-decode was that a deprecaterd way of importing the package (`import * as jwt_decode from 'jwt-decode'`) has been removed. We were already using the preferred import syntax in the only place we use this library, namely in [`user.js`](https://github.com/gordon-cs/gordon-360-ui/blob/develop/src/services/user.js#L7), so this should not cause any problems. Considering that the decoding is still successful (i.e. the [getLocalInfo()](https://github.com/gordon-cs/gordon-360-ui/blob/develop/src/services/user.js#L362) method and all it's uses), this is a clean update.